README.md

```diff
@@ -191,6 +191,44 @@ For more information on Translating with Sphinx, see [Sphinx i18n].
 * Add translated html output for all available languages, see [i18n]
 * Run `fab pdf` to produce PDF output for distribution
 
+## Troubleshooting
+
+### Linux troubleshooting
+ON some systems, using the system package version of pip and running the command `pip install -r requirements.txt` might damage the python installation. Concretely, pyopenssl might get damaged and you might get an error saying `'module' object has no attribute 'SSL_ST_INIT'`
+
+This happens because the repository has an old version and when installing/updating the requirements, the scripts break.
+
+The solution to fix this consists on deleting the broken files, remove the installed package and install it manually (all commands require access privileges, so use sudo or whatever you need)
+Fix python:
+```
+rm -rf /usr/lib/python2.7/dist-packages/OpenSSL
+rm -rf /usr/lib/python2.7/dist-packages/pyOpenSSL-0.15.1.egg-info
+```
+Remove pip and repair python just in case:
+```
+apt-get purge python-pip
+apt-get install --reinstall python
+```
+Install pip manually:
+```
+curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+python get-pip.py
+```
+
+Alternatively, instead of installing pip on the system, you can also install a python-virtualenv and then use PyPI in that virtualenv.
```

Comment:
I think we should probably make this part of the instructions -- I never do anything outside of a virtualenv, but I didn't want to complicate the instructions.

Reply:
I've never worked with a virtualenv, so I'm not the best person to write that then.